client/tx/search.go

```diff
@@ -102,17 +102,44 @@ func SearchTxRequestHandlerFn(ctx context.CoreContext, cdc *wire.Codec) http.Han
 		tag := r.FormValue("tag")
 		if tag == "" {
 			w.WriteHeader(400)
-			w.Write([]byte("You need to provide a tag to search for."))
+			w.Write([]byte("You need to provide a tag or an address to search for."))
```

Comment:
Change this error - maybe "You need to provide at least `key=value` pair to search for. Postfix the key with `_bech32` to search bech32-encoded addresses or public keys".